docs/source/01_intro.md

```diff
@@ -1,5 +1,48 @@
 # Introduction
 
+ADAM is a library and command line tool that enables the use of [Apache
+Spark](https://spark.apache.org) to parallelize genomic data analysis across
+cluster/cloud computing environments. ADAM uses a set of schemas to describe
+genomic sequences, reads, variants/genotypes, and features, and can be used
+with data in legacy genomic file formats such as SAM/BAM/CRAM or VCF, as well
```

Comment:
I have draft mapping documentation for the formats here
https://github.com/heuermh/bdg-formats/tree/docs/docs/source

We don't currently host docs or javadocs for bdg-formats, so I hadn't found the motivation to complete them.  Would those make sense in the ADAM docs?